### ISSUE TITLE
All mobs from a mod

### ISSUE THREAD

**Issue by Tw1L1ght12** [holmraven/ChickensShed#8]
Is it possible to add all mobs from a mod (ex. exotic birds) via config in one string?
Like:
`"exoticbirds:all" = "minecraft:feather"`

**Comment by holmraven** [comment]
> Is it possible to add all mobs from a mod (ex. exotic birds) via config in one string? Like: `"exoticbirds:all" = "minecraft:feather"`

No, it is not possible as of right now, you would have to enter every mob separately. I have not thought about this before, but it certainly would be possible for me to implement a new config option, something like a mod whitelist, in the future.
